### ISSUE TITLE
Feature Suggestion: Improve Debugging

### ISSUE THREAD

**Issue by bigtimebuddy** [kayahr/jest-electron-runner#14]
Hello, I've been trying to transition my project away from jest-electron because of incompatibility with the latest Jest and because of old Electron version supported. 

This project has worked very well except for debugging. 

Using the steps `getCurrentWindow().show();` simply opens a blank window. I'm not able to use `debugger` statements and I don't see any Jest output on the screen.

jest-electron uses an environment variable, e.g., `DEBUG_MODE=1 jest` which will show the electron window and allow for breakpoints, output Jest to the DOM, etc. I like the environment variable because it makes it easy to invoke and turn on or off debugging without having to change the source code of any tests.

**Comment by semireg** [comment]
I agree. My workflow would really benefit from being able to debug with DevTools, console, etc.

**Comment by kayahr** [comment]
I'm closing this issue because there will be no fix. I'm no longer using Jest, switched to Vitest and I'm going to archive this project. If someone wants to take over, feel free to fork it.
